src/EditorFeatures/VisualBasic/NavigationBar/VisualBasicNavigationBarItemService.vb

```diff
@@ -65,23 +67,23 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.NavigationBar
             Return TypeOf item IsNot AbstractGenerateCodeItem
         End Function
 
-        Private Function GetTypesInFile(semanticModel As SemanticModel, cancellationToken As CancellationToken) As IEnumerable(Of INamedTypeSymbol)
+        Private Function GetTypesAndPositionsInFile(semanticModel As SemanticModel, cancellationToken As CancellationToken) As IEnumerable(Of Tuple(Of INamedTypeSymbol, Integer))
```

Comment:
Yeah, that's why I tried to ensure everywhere this Tuple was used/returned it used the descriptive name "typesAndPositions" but I see I've missed one case below. I'll fix that.

I guess returning the SyntaxNode is *slightly* better, but even seeing a Tuple<INamedTypeSymbol, SyntaxNode> in this code wouldn't really let you know what's going on -- is it the type where we're generating the handlers, or the type containing the events (different in WithEvents cases)? Etc. Still, it's enough of an improvement that I think I'll take the advice. :smile: